dasharo-compatibility/watchdog.robot

```diff
@@ -135,3 +135,35 @@ WDT005.001 Watchdog is detected by OS (Ubuntu 22.04)
         ${platform_has_reset}=    Set Variable    ${FALSE}
     END
     Should Be Equal    ${platform_has_reset}    ${FALSE}
+
+WTD006.001 Watchdog resets platform on kernel crash (Ubuntu 22.04)
+    [Documentation]    Boot into OS with the watchdog enabled, crash the kernel
+    ...    and verify that the watchdog resets the machine.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    APU002.001 not supported
+    Skip If    not ${WATCHDOG_SUPPORT}    Watchdog tests not supported.
+    Power On
+    ${setup_menu}=    Enter Setup Menu Tianocore And Return Construction
+    ${dasharo_menu}=    Enter Dasharo System Features    ${setup_menu}
+    ${chipset_menu}=    Enter Dasharo Submenu    ${dasharo_menu}    Chipset Configuration
+    Set Option State    ${chipset_menu}    Enable watchdog    ${TRUE}
+    # Refresh menu, in case the watchdog timeout wasn't already available
+    ${chipset_menu}=    Reenter Menu And Return Construction
+    Set Option State    ${chipset_menu}    Watchdog timeout value    300
+    Save Changes And Reset
+    Boot System Or From Connected Disk    ubuntu
+    Login To Linux
+    Switch To Root User
+    # Verify that there are entries from wdt or watchdog in kernel logs
+    ${out}=    Execute Linux Command    dmesg | grep -E 'watchdog|wdt'
```

Comment:
Point 4 in: https://github.com/Dasharo/open-source-firmware-validation?tab=readme-ov-file#keywords

Reply:
@macpijan Why is `Execute Linux Command` wrong here? We use it in many tests after logging into Linux.

Reply:
Because we want to achieve a goal of reducing keyword duplication. Adding new tests using duplicated keywords will not help here.